scripts/build_rdkitjs.sh

```diff
@@ -47,6 +47,9 @@ cp docs/GettingStartedInJS.html $MINIMALLIB_OUTPUT_PATH/GettingStartedInJS.html
 echo "Build completed"
 echo "MinimalLib distribution files are at $MINIMALLIB_OUTPUT_PATH"
 
+# Move typescript files to dist folder
+cp typescript/index.d.ts $MINIMALLIB_OUTPUT_PATH/index.d.ts
```

Comment:
this makes sure to include the types to the package dist files during the build of the RDKit module